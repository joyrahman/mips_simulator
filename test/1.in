ADDI R1 R0 10
ADDI R5 R0 2
ADDI R2 R0 196
ADDI R3 R0 396
ST R1 R2 0
MUL R4 R1 R5 
ST R4 R3 0
SUBI R2 R2 4
SUBI R3 R3 4
SUBI R1 R1 1
BNEQZ R1 -7
ADDI R1 R0 10
ADDI R2 R0 196
ADDI R3 R0 396
ADDI R4 R0 596
LD R5 R2 0
LD R6 R3 0
ADD R5 R5 R6
ADDI R7 R0 4
DIV R5 R5 R7
ST R5 R4 0
SUBI R2 R2 4
SUBI R3 R3 4
SUBI R4 R4 4
SUBI R1 R1 1
BNEQZ R1 -11
HLT 
