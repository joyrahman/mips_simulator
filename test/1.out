189
189
354
165
0
156
356
556
0
2
4
